src/pages/Cart/Cart.js

```diff
@@ -1,9 +1,53 @@
 import React, { Component } from 'react';
+import ShoppingBag from './CartPage/ShoppingBag';
+import Delivery from './CartPage/Delivery';
+import Payment from './CartPage/Payment';
 import './Cart.scss';
 
 class Cart extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cartListData: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('./data/cartListData-doyoung.json')
```

Comment:
```suggestion
    fetch('/data/cartListData-doyoung.json')
```
상대경로를 적으면 현재 위치한 routing 기준으로 접근하기 때문에 원하는대로 동작하지 않을 수 있습니다. 항상 루트에서부터 접근할 수 있도록 절대경로로 시작해주세요

Reply:
🙆‍♂️수정완료하였습니다